py/desispec/scripts/fluxcalibration.py

```diff
@@ -172,26 +175,30 @@ def main(args) :
         model_fibers=model_fibers[ok]
         model_metadata=model_metadata[:][ok]
 
+    stdcheck = not args.nostdcheck
+        
     # check that the model_fibers are actually standard stars
     fibermap = frame.fibermap
 
     ## check whether star fibers from args.models are consistent with fibers from fibermap
     ## if not print the OBJTYPE from fibermap for the fibers numbers in args.models and exit
-    fibermap_std_indices = np.where(isStdStar(fibermap))[0]
-    if np.any(~np.in1d(model_fibers%500, fibermap_std_indices)):
-        target_colnames, target_masks, survey = main_cmx_or_sv(fibermap)
-        colname =  target_colnames[0]
-        for i in model_fibers%500:
-            log.error("inconsistency with spectrum {}, OBJTYPE={}, {}={} in fibermap".format(
+    if stdcheck:
+        fibermap_std_indices = np.where(isStdStar(fibermap))[0]
+        if np.any(~np.in1d(model_fibers%500, fibermap_std_indices)):
+            target_colnames, target_masks, survey = main_cmx_or_sv(fibermap)
+            colname =  target_colnames[0]
+            for i in model_fibers%500:
+                log.error("inconsistency with spectrum {}, OBJTYPE={}, {}={} in fibermap".format(
                 i, fibermap["OBJTYPE"][i], colname, fibermap[colname][i]))
         sys.exit(12)
```

Comment:
Lesson learned: always run test case before merging.  This sys.exit(12) should have been indented under the `if np.any(~np.in1d(model_fibers%500, fibermap_std_indices))` check, and caused the pipeline to exit early last night.  Credit to @akremin for debugging that.

Reply:
Oops. My apologies for missing that/not testing other than running pytest.